vimdiff: cleanup and update page (#4968)
tldr-pages/tldr: pages/common/vimdiff.md

# vimdiff

> Open up two or more files in vim and show the differences between them.

> > More information: https://www.vim.org.

Reviewer: Maybe we could add a see also here too?
```suggestion
> See also `vim`.
> More information: https://www.vim.org.
```

Reviewer 2: ```suggestion
> More information: <https://www.vim.org>.
```

- Open two files and show the differences:

`vimdiff {{file1}} {{file2}}`

- Move the cursor to the window on the left|right:

`Ctrl + w {{h|l}}`

> - Jump to next difference:

Reviewer: Perhaps this will read better?
```suggestion
- Jump to the next difference:
```

`[c`

> - Jump to previous difference:

Reviewer: Perhaps this will read better?
```suggestion
- Jump to the previous difference:
```

`]c`

> - Obtain difference from other window to current window:

Reviewer: Perhaps this will read better?
```suggestion
- Copy differences from another window to the current window:
```

Author: This would actually be incorrect- only the highlighted difference is copied/obtained, and "another" is a bit confusing as this tldr assumes only two windows, so "other" would be a bit more clear than "another". My edit takes these points into consideration.

Reviewer: Not sure I'm following you, but I'm not a vim user lol :P

Anyway, the point I was trying to get at here is that `Obtain` sounds very odd indeed here. Obtain generally means to "gain possession of something", which I somewhat doubt is the meaning you're after here. Is there a way we can reword this at all?

Author: Sorry I might not have been clear... I agree, "obtain" does sound a bit odd even though technically that word was [selected by the Vim author](http://vimdoc.sourceforge.net/htmldoc/diff.html#do) to match with the shortcut letter "o", so I have changed the description it mostly based on your suggestion since tldr should be intuitive for beginners. You can see that change in [my most recent update to the PR](https://github.com/tldr-pages/tldr/pull/4968/files).

Reviewer: Ah, I see. I didn't realise that `Obtain` was a technical term. In that case, what about this?

```suggestion
- Obtain differences from other window in the current window:
```

Author: Unfortunately, that is still incorrect. This command only grabs one difference, not all of them.

Do you not like [my most recent commit wording](https://github.com/tldr-pages/tldr/pull/4968/files) on line 23?
`- Copy the highlighted difference from the other window to the current window:`

If you like the technical term, I'm happy to modify it to:
`- Obtain the highlighted difference from the other window to the current window:`

Reviewer: Ah, I didn't notice that you'd changed it. Thanks!

I think it's fine with the new version.

`do`

> - Put difference from current window to other window:

Reviewer: Perhaps this will read better?
```suggestion
- Copy the differences from the current window to another window:
```

Author: See my above response.

`dp`

- Update the diff highlighting and folds:

`:diffupdate`

> - Toggle code fold:

Reviewer: Perhaps this will read better?
```suggestion
- Toggle code folding:
```

Author: This would also be incorrect since it only toggles the highlighted code fold, not all folds. I've updated this line in my new commit to make that more clear however.

# vimdiff

> Open up two or more files in vim and show the differences between them.
> See also `vim`.

> > More information: https://www.vim.org.

Reviewer 2: ```suggestion
> More information: <https://www.vim.org>.
```
